pom.xml

```diff
@@ -30,10 +30,10 @@ under the License.
 
     <groupId>org.apache.flink</groupId>
     <artifactId>flink-shaded</artifactId>
-    <version>1</version>
+    <version>1.0</version>
```

Comment:
Mh. Okay. I wonder how often we need to address bugs using minor versions in the parent pom.

This this, we'll end up with a complicated versioning scheme for example for guava: `1.2-19.0`, "1-19.0" looks a little bit easier.

But I'm okay with merging it like this.